nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -493,6 +559,11 @@ def __call__(self, dataset: DocumentDataset):
         minhashLSH = Sequential([self.minhash, self.lsh])
         buckets_df = minhashLSH(dataset)
         print(f"Stage{stage_num}: Minhash + LSH complete!")
+        if buckets_df is None:
+            print(
+                f"Stage{stage_num}: No potential duplicate documents found during LSH"
+            )
+            return None
```

Comment:
Should this return None or an empty `DocumentDataset` with no id's

Reply:
I prefer returning `None`. Empty DocumentDatasets might lead to unexplained errors downstream that might be tougher to debug/understand. Happy to hear counter points. 
One thing that comes up from this is that I might update the `examples/FuzzyDedup.py` to handle the case where the result returned was `None`

Reply:
Makes sense, but then for `Sequential` I think we might want to handle that behavior too?

Reply:
I haven't seen `Sequential` being used directly with `FuzzyDuplicates` since the results cannot be processed downstream by any of the other modules without using to filter out the duplicates first. I'm not sure how to handle this use case. But longer term, we would probably want to add a `FuzzyDeduplicate` class that calls Fuzzy Duplicates and also handles removal.